spec/views/organizations/index.html.erb_spec.rb

```diff
@@ -3,12 +3,47 @@
 require 'rails_helper'
 
 RSpec.describe 'organizations/index', type: :view do
+  let(:org1) { create(:organization, name: 'Organization 1') }
+  let(:org2) { create(:organization, name: 'Organization 2') }
+  let(:org3) { create(:organization, name: 'Organization 3', provider: false) }
+
   before do
-    assign(:organizations, [create(:organization, name: 'Name1'), create(:organization)])
+    assign(:organizations, [org1, org2, org3])
   end
 
   it 'renders a list of organizations' do
     render
-    assert_select 'tr>td', text: 'Name1'.to_s
+
+    assert_select 'tr>td', text: 'Organization 1'
+    assert_select 'tr>td', text: 'Organization 2'
+    assert_select 'tr>td', text: 'Organization 3'
```

Comment:
We know providers shouldn't show up in this list and there is a ticket to address it; the test should change with that PR? 
I just wanted to verify the current behavior, which also lined up with the doc.
Can remove if it's not good to add.

Reply:
PR #735 fixes this issue; note to change the test to reflect that once merged.

Reply:
I think it's okay to add this. I'll fix the test before #735 is merged.